lib/containers/common.pm

```diff
@@ -262,4 +262,39 @@ sub test_rpm_db_backend {
     }
 }
 
+sub check_containers_connectivity {
+    my $runtime = shift;
+    record_info "connectivity", "Checking that containers can connect to the host, to each other and outside of the host";
+    my $container_name = 'sut_container';
+
+    # Run container in the background
+    assert_script_run "$runtime pull " . registry_url('alpine');
+    assert_script_run "$runtime run -id --rm --name $container_name -p 1234:1234 " . registry_url('alpine') . " sleep 30d";
```

Comment:
just making sure. did you really meant 30 days? :)

Reply:
Yes as the `infinite` value does not work on `sleep` from `busybox`. :smile:

Reply:
Please add a corresponding comment e.g. `30d because infinite is not supported by sleep in busybox`